Source/DafnyCore/Triggers/SplitPartTriggerWriter.cs

```diff
@@ -185,12 +185,12 @@ string InfoFirstLineEnd(int count) {
       errorReporter.Message(MessageSource.Rewriter, warningLevel, null, reportingToken,
         "Could not find a trigger for this quantifier. Without a trigger, the quantifier may cause brittle verification. " +
         $"To silence this warning, add an explicit trigger using the {{:trigger}} attribute. " +
-        "For more information, see the section quantifier instantiation rules in the reference manual.");
+        "For more information, see the section on quantifier instantiation rules in the reference manual.");
     } else if (!CouldSuppressLoops && !AllowsLoops) {
       errorReporter.Message(MessageSource.Rewriter, warningLevel, null, reportingToken,
         "Triggers were added to this quantifier that may introduce matching loops, which may cause brittle verification. " +
         $"To silence this warning, add an explicit trigger using the {{:trigger}} attribute. " +
-        "For more information, see the section quantifier instantiation rules in the reference manual.");
+        "For more information, see the section on quantifier instantiation rules in the reference manual.");
```

Comment:
There shouldn't be a `$` here (and hence no doubled `{{`). Would be nice to extract this duplicated message out.